src/css/palette.styl

```diff
@@ -21,10 +21,11 @@ orange-core = hsl(19, 75%, 55%) // Nisei Anarch
 
 // Golds
 //Mainly Jinteki.net button styles
-gold-dark = hsl(19, 56%, 40%)
```

Comment:
Is this something you need to update in the rest of the code?

Reply:
nowhere else uses gold-dark (that I could find) but I found the current value of gold-dark was more of a weird orange I decided to change it. That being said, I didn't even end up using this new value in my code so it might not be worth changing.